content/docs/learn/quickstart/index.md

```diff
@@ -231,6 +231,12 @@ The Arrow team follows an eager dependency update policy. That means that Arrow
 
 :::
 
+### Additional setup for Android
+
+If you are using Arrow in Android, you should enable
+[library desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) in your final application.
```

Comment:
Is that also needed for the other modules? What are the benefits in those cases?